package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "graphqlsp",
-  "version": "0.2.0",
+  "name": "@0no-co/graphqlsp",
```

Comment:
This rebase seems to have been a bit corrupting 😅

Reply:
Oh shucks! Should have double-checked 😅